src/kit.ts

```diff
@@ -603,13 +603,19 @@ const MSVC_ENVIRONMENT_VARIABLES = [
   'LIBPATH',
   'NETFXSDKDir',
   'Path',
+  //'Platform', - disabled at the request of @bobbrow as this causes some of MS internal projects to fail to build
```

Comment:
We don't normally annotate our code like this.  I think it is sufficient to say that it is disabled because it is currently unnecessary and causes some projects to fail to build.

Reply:
Changed as requested.